src/main/java/Storage.java

```diff
@@ -0,0 +1,87 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.ArrayList;
+import java.nio.charset.Charset;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+
+public class Storage {
+    private String directory = System.getProperty("user.home");
+    private String fileName = "tasks.txt";
+    private Path path = Paths.get(directory, "Desktop", "duke", "data", fileName);
+
+    private ArrayList<Task> tasks;
+
+    public Storage() {
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Writes to storage in hard disk to store tasks added.
+     *
+     * @param content data that is to be written to the file.
+     * @throws IOException  If invalid path or file.
+     */
+
```

Comment:
You include an extra newline before the method here, but exclude a newline before the method in other classes. You may want to fix this inconsistency.